src/components/Security/Roles/CreateOrUpdate.vue

```diff
@@ -97,6 +99,28 @@
   </b-container>
 </template>
 
+<style lang="scss" scoped>
```

Comment:
Styles duplicated with that of the `profiles/createOrUpdate` component.
Though I don't know if it's a good thing to refactor it here and now